docs/operations.md

```diff
@@ -0,0 +1,57 @@
+# Operations
+
+`PhoenixSwagger` provides `swagger_path/2` macro that generates swagger specification
+for the certain phoenix controller action.
+
+Example:
+
+```elixir
+use PhoenixSwagger
+
+swagger_path :index do
+  get "/posts"
+  description "List blog posts"
+  response 200, "Success"
+end
+
+def index(conn, _params) do
+  posts = Repo.all(Post)
+  render(conn, "index.json", posts: posts)
+end
+```
+
+The `swagger_path` macro takes two parameters:
+
+* Name of controller action;
+* `do` block that contains the `swagger` specification for the controller action.
+
+Within the do-end block, the DSL provided by the `PhoenixSwagger.Path` module can be used.
+The DSL usually starts with one of the `get`, `put`, `post`, `delete`, `head`, `options` functions,
```

Comment:
I have errors with `delete` path. Required to use in code `PhoenixSwagger.Path.delete ...` - maybe add a note about it?